pkg/controller/job/job_controller.go

```diff
@@ -188,8 +196,9 @@ func (jm *Controller) Run(ctx context.Context, workers int) {
 	defer jm.queue.ShutDown()
 	defer jm.orphanQueue.ShutDown()
 
-	klog.Infof("Starting job controller")
-	defer klog.Infof("Shutting down job controller")
+	logger := klog.LoggerWithName(klog.FromContext(ctx), "Job")
```

Comment:
Why to wrap the logger here by `klog.LoggerWithName`, but not in in other places within the job_controller?

Reply:
Please take out `LoggerWithName`. We plan to add the names one level up in the caller of this function.